src/sentry/api/serializers/rest_framework/dashboard.py

```diff
@@ -195,18 +182,25 @@ def validate(self, data):
                 "organization_id": self.context.get("organization").id,
             }
 
-            snuba_filter = get_filter(conditions, params=params)
+            builder = UnresolvedQuery(
+                dataset=Dataset.Discover,
+                params=params,
+                equation_config={"auto_add": not is_table, "aggregates_only": not is_table},
+            )
+
+            builder.resolve_conditions(conditions, use_aggregate_conditions=True)
+            builder.resolve_params()
```

Comment:
Should we do a full `builder.get_snql_query().validate()` here? Or is that overkill? 🤔 would require the `QueryBuilder` instead of the `UnresolvedQuery` class

Reply:
actually, that might be better, let me take a look!

Reply:
order by validation breaks rn